robot/src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,25 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+/**
+ * Constants for the robot
+ * 
+ * Includes CAN IDs for devices, as well as tuning parameters for closed-loop
+ * control
+ */
+public class Constants {
+    // --- Drivetrain Constants ---
+    // We are using CAN based controllers. Each controller has an ID associated with it
+    // We use these IDs to directly communicate with a selected device over CAN bus
+    public static final int kDrivetrainLeftMasterId = 0;
```

Comment:
Let's consider subclassing these for context when using. Example [here](https://github.com/itrt4176/deepspace2019/blob/master/src/main/java/com/irontigers/RobotMap.java#L26) `RobotMap.XBoxController. LEFT_X_AXIS`.